doc/connectivity/parcellating.rst

```diff
@@ -124,13 +121,8 @@ Using and visualizing the resulting parcellation
 Visualizing the parcellation
 -----------------------------
 
-For every scikit-learn clustering object, the labels of the parcellation
-are found its `labels_` after fitting it to the data. To turn them into a
-brain image, we need to unmask them with the :class:`NiftiMasker`
-*inverse_transform* method.
-
-Note that by default, clusters are labeled from 0 to
-(n_clusters - 1), and the label 0 may be confused with a background.
+The labels of the parcellation are found its `labels_img_` after fitting it to the data
```

Comment:
are found in the `labels_img_` attribute of the :class:`...Parcellation` object after fitting it to the data.